README.md

```diff
@@ -239,10 +246,10 @@ If your extension requires one or more of those plugins to be present, make sure
 The Hide Tribe Bar extension requires that The Events Calendar 3.0 or greater is present. This line is the only one needed to specify that:
 
 ```
-$this->add_required_plugin( 'Tribe__Events__Main', '3.0' );
```

Comment:
The next paragraph explains it in more detail:

> Note: All extensions require Tribe Common 4.3.3 or greater to be active. The Events Calendar 3.0 did not include that library. It would take exotic circumstance for an extension to run on an older version like 3.0. For example, if Event Tickets 4.3.3 was active and working properly, then this version comparison would run. If Event Tickets was not active, nor any other plugin that includes Tribe Common, the Extension would cease running even before a version comparison was made.

As it says, it Event Tickets 4.3.3 and The Events Calendar 3.0 were active and working together, then the extension would run fine. But if you change this line to `4.3` then the extension will no longer work on this old version of The Events Calendar. To me there is no reason to make it not work on older versions when we don't have to.

Perhaps we just need to clarify that paragraph. If it was unclear to you, it's probably nor clear to others. But I would still like to communicate this just so people understand how it works.